## Pathogenesis {.page_break_before}

### Mechanism of Host Infection by SARS-CoV-2

This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.

#### Primary Transmission and Viral Entry

[How does SARS-CoV-2 enter human cells?]
[What cells are primary infection sites for SARS-CoV-2?]
[What structural aspects allow for viral entry]?

#### Viral Replication and Spreading

[Renamed section with transmission dynamics below] 
[Basic introduction into replication cycle]
[What are the routes of transmission]

#### Dynamics of Transmission 
Accurate estimates of the reproduction number of COVID-19 are crucial to understanding dynamics of infection and predicting the effects of interventions. The basic reproduction number, R0, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [cite]. The effective reproduction number, R(t), describes the reproduction number at a specific time, and is used to measure changes in R over time, as different fractions  of the population become infected, or as interventions are put into place. R0 and R(t) can be estimated directly from epidemiological data or inferred using mathematical models (typically susceptible-infected-recovered (SIR) models and their extensions) [@doi:10.1126/science.272.5258.74]. 

R0 for COVID-19 is estimated to lie in the range R0=1.4-6.5 [@doi:10.1093/jtm/taaa021, @doi:10.1101/2020.03.20.20038422; @doi:10.1016/j.ijid.2020.02.033], where estimates vary widely depending on the region and the methods used. Estimates currently derive from populations in Asia (e.g. China, Iran) since outbreaks in Europe and North America are more recent. Data-derived estimates, i.e. those not using SIR-type models, typically predict lower values of R0, with R0=2.0-3.6 in China (Wuhan and other regions), 
[Majumder, Liu, Joseph]
and R0=2.06-2.52 on a cruise ship where an outbreak occurred [@doi:10.1016/j.ijid.2020.02.033].
SIR model-derived estimates of R0 range from 2.0 - 6.5 in China [Read, Shen, Cao, Tang] and R0=4.86 in Iran [@doi:10.1101/2020.03.20.20038422]. From meta-analysis of studies estimating R0, [@doi:10.1093/jtm/taaa021] predict the median as R0=2.79.

Inference of the time-dependent reproduction number can provide insight into how populations are responding to an infection, and the effectiveness of interventions. In China, R(t) was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10.1016/S1473-3099(20)30144-4]. R(t) decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after. Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%. In South Korea, R(t) was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@url:https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf]. Metro data was also analyzed to estimate the effects of social distancing measures. R(t) decreased in Daegu from around 3 to below 1, over the period that social distancing measures were introduced. In Seoul, R(t) decreased slightly, but remained close to 1 (greater than in Daegu). This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul R(t) remains above 1, thus secondary outbreaks are possible. It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do. 


Other important messages -effects on contact tracing, social distancing, importnace of region specificity in reporting and tracking 


### Immune Response to SARS-CoV-2

[Cellular responses to SARS-CoV-2 infection]
[What is causing neutropenia and lymphopenia observed in COVID-19 patients]
[Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover]
[Cytokines and other soluble factors contribution to immune response]

### Systems level approaches for understanding SARS-CoV-2 pathogenesis

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].

The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7]. 
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7] or from BALF, throat swabs, or isolates of the virus cultured from BALF [@doi:10/ggjr43]. 
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43]. 
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].

The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis. 
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently [@doi:10.1126/science.abb2507]. 
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1]. 
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.

There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052]. 
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074]. 
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
