## Pathogenesis {.page_break_before}

### Mechanism of Host Infection by SARS-CoV-2

This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.

#### Primary Transmission and Viral Entry

[How does SARS-CoV-2 enter human cells?]
[What cells are primary infection sites for SARS-CoV-2?]
[What structural aspects allow for viral entry]?

#### Viral Replication, Spreading and Transmission

[Basic introduction into replication cycle]
[What is the basic reproductive rate]
[What are the routes of transmission]

### Immune Response to SARS-CoV-2

[Cellular responses to SARS-CoV-2 infection]
[What is causing neutropenia and lymphopenia observed in COVID-19 patients]
[Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover]
[Cytokines and other soluble factors contribution to immune response]

### Systems level approaches for understanding SARS-CoV-2 pathogenesis

[Sandipan Ray volunteered to lead this section]
Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. 
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases (Hood and Tian, 2012, PMID: 23084773). 
Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases (Eckhardt et al., 2020, PMID: 32060427). 
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections (Ray et al., 2014, PMID: 24293340).
The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen (Wu et al., Nature 2020, PMID: 32015508; Lu et al., Lancet. 2020, PMID: 32007145, and Zhou et al., 2020, PMID: 32015507). 
Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) (Lu et al., Lancet. 2020, PMID: 32007145). 
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) indicating its possible bat origin (Lu et al., Lancet. 2020, PMID: 32007145; Zhou et al., 2020, PMID: 32015507). 
The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.  
In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, proving some clue regarding which part of the pathogen is recognized by human immune responses (Grifoni et al., 2020, PMID: 32183941). 
Importantly, overactivation of T cells, caused by an increase of Th17 and high cytotoxicity of CD8 T cells, could be one of the major reasons behind the severe immune injury and fatality associated with SARS-CoV-2 infection (Xu et al., 2020, PMID: 32085846). 
Availability of the genome sequence of SARS-CoV-2 enhanced the possibilities for the subsequent proteome level studies that can provide further mechanistic insights into its complex pathogenesis. 
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, is reported very recently (Wrapp et al., 2020, PMID: 32075877). 
Even though any comprehensive proteomic analysis of the pathogen or the patients suffering from its infection has not been reported yet, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Cacoâ€“2 cells as an infection model (Bojkova et al., 2020, doi:10.21203/rs.3.rs-17218/v1).  
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
There is a high level of sequence homology between SARS-CoV-2 with SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry (Hoffmann et al., 2020, PMID: 32142651). 
Consequently, earlier proteome level studies on SARS-CoV can also provide some essential information regarding the new pathogen (Chen et al., 2004, PMID: 15572443; He et al., 2004, PMID: 15020242). 
Considering the paucity of omics-level big data sets on for SARS-CoV-2 till now, the existing data hubs that contain adequate information for the other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), The Virus Pathogen Resource (ViPR) could be the useful resources for computational and bioinformatics research on SARS-CoV-2. 
